transport/sctp/SctpAssociationImpl.cpp

```diff
@@ -184,7 +184,7 @@ size_t SctpAssociationImpl::GenericCookie::maxSize() const
 void SctpAssociationImpl::GenericCookie::set(const ChunkParameter& param)
 {
     std::memcpy(cookie, param.data(), std::min(static_cast<size_t>(param.dataSize()), sizeof(cookie)));
-    length = param.length.get();
+    length = param.dataSize();
```

Comment:
bug fix for SCTP establishment

Reply:
Nice, I have been some sporadic failures to open datachannels and I added logs around the Cookie negation. Perhaps this was the reason